.travis.yml

```diff
@@ -1,40 +1,46 @@
 language: php
 
-cache: vendor
```

Comment:
Will did to add this back once https://github.com/travis-ci/travis-ci/issues/3563 will be solve